content/docs/command-reference/run.md

```diff
@@ -93,6 +96,14 @@ data pipeline (e.g. random numbers, time functions, hardware dependency, etc.)
   > Note that a DVC-file without dependencies is considered always changed, so
   > `dvc repro` always executes it.
 
+- `-p [<filename>:]<params_list>`, `--params [<filename>:]<params_list>` -
+  specify a subset of parameters from a parameter file the stage depends on. The
+  params subset can be specified by coma separated params list:
+  `-p learning_rate,epochs`. By default, the params file is `params.yaml` but
+  this value can be redefined with params prefix:
+  `-p parse_params.yaml:threshold` See `dvc params` to learn more about using
+  parameters.
```

Comment:
```suggestion
  specify a subset of [experiment parameters](/doc/command-reference/metrics) the
  stage depends on, from a parameter file. This is done by sending a coma
  separated list as argument, e.g. `-p learning_rate,epochs`. By default, the
  parameters file name is `params.yaml`, but this can be redefined with a prefix
  in the argument to this option, e.g. `-p parse_params.yaml:threshold`. See
  `dvc params` to learn more about parameters.
```